### PR TITLE
div. Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy [hunk 1]
```diff
@@ -224,6 +224,20 @@ class CreateController {
...
+                if(user.isAdmin() || user.getSuperUserStatus()) {
+                  flash.message = "Object was not assigned to a curator group because you are admin or superuser!!!!"
+
+                }else {
+                  user.curatoryGroups.each { CuratoryGroup cg ->
+                    result.newobj.curatoryGroups.add(cg)
+                  }
+                }
+
+                  result.newobj.save(flush:true)
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy:238]: try to avoid flush:true whenever possible

Comment (croqueGrec09) [reply server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy:238]: arranged in personal conversation - flushes will be dealt in separate refactoring

File: server/gokbg3/grails-app/migrations/changelog-2021-03-30.groovy [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+        createIndex(indexName: "ti_medium_idx", tableName: "title_instance") {
+            column(name: "medium_id")
+        }
+    }*/
+
+    changeSet(author: "djebeniani (modified)", id: "1617095504908-87") {
+        grailsChange {
+            change {
+
+                sql.execute("update package set pkg_scope_rv_fk = null where pkg_scope_rv_fk = (select rdv_id from refdata_value where rdv_owner = (SELECT rdc_id FROM refdata_category WHERE rdc_description = 'Package.Scope') and rdv_value = 'Aggregator')")
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/migrations/changelog-2021-03-30.groovy:459]: for better retracability, use one changeset per query

File: server/gokbg3/src/test/groovy/wekb/GlobalSearchTemplatesServiceSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package wekb
+
+import grails.testing.services.ServiceUnitTest
+import spock.lang.Specification
+
+class GlobalSearchTemplatesServiceSpec extends Specification implements ServiceUnitTest<GlobalSearchTemplatesService>{
```
Comment (croqueGrec09) [inline server/gokbg3/src/test/groovy/wekb/GlobalSearchTemplatesServiceSpec.groovy]: the test should not have been checked in ...

Comment (Moetez16) [reply server/gokbg3/src/test/groovy/wekb/GlobalSearchTemplatesServiceSpec.groovy]: Deleted

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**